modules/shared/components/atoms/VoteImage/VoteImage.tsx

```diff
@@ -0,0 +1,36 @@
+import type { ReactElement, FC } from 'react';
+import React from 'react';
+import type { IVoteImage } from './IVoteImage';
+import OutlinedHeartIcon from '../../icons/outlinedHeart.svg';
+import FilledHeartIcon from '../../icons/filledHeart.svg';
+import OutlinedDislikeIcon from '../../icons/outlinedDislike.svg';
+import OutlinedLikeIcon from '../../icons/outlinedLike.svg';
+import FilledLikeIcon from '../../icons/filledLike.svg';
+import styles from './VoteImage.module.css';
+
+const VoteImage: FC<IVoteImage.IProps> = ({ oneImage }): ReactElement => {
```

Comment:
- I think it would be more precise if we name the component `VotingIcon`. 
- For someone who read the code for the first time => he may ask himself  `oneImage`  prop: what does it mean  ? 🤔  . does it mean to show one image, it is value will be `img url` ?
try to read this for Ex: `isOneImageVote`?. do you think is there a difference.

Reply:
- I agree with you.
- I thought about that at first, can't remember what made me change my mind :).

Reply:
Good job 👏🎉. We fisrt make it work then reiterate and make improvements. 
By the naming is very difficult but it will make the code self documented and sharing the code with any one will help him understand it and help you faster in case there is a bug.
One more benefit, it will help the reader understand the bussiness behind the component implicitly even if he has not any idea a bout the project.